### ISSUE TITLE
v2 ansible-playbook should print the help message if run without parameters

### ISSUE THREAD

**Issue by kustodian** [ansible/ansible#11123]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible 2.0.0 (devel 620fad9f8d) last updated 2015/06/02 12:21:42 (GMT +200)
##### Ansible Configuration:

N/A
##### Environment:

Ubuntu 15.04
##### Summary:

When you run `ansible-playbook` without parameters it prints an error message like this:

```
$ ansible-playbook 
Traceback (most recent call last):
  File "/home/kustodian/git/ansible/bin/ansible-playbook", line 65, in <module>
    cli.parse()
  File "/home/kustodian/git/ansible/lib/ansible/cli/playbook.py", line 73, in parse
    raise AnsibleOptionsError("You must specify a playbook file to run")
NameError: global name 'AnsibleOptionsError' is not defined
```

On older version it printed help (`--help`), but what is also important is that the `ansible` command (even in v2) still prints help when run without any parameters, so I guess this is a bug for `ansible-playbook`.

**Comment by my-janala** [comment]
I am using version 1.9 and it does that print help as the reporter mentioned.

**Comment by bcoca** [comment]
try now, fixed via 47be5b416658ef1474aee89873fbd72622f83777

**Comment by kustodian** [comment]
Works fine now. Thanks.
